packages/medusa/src/api/routes/admin/products/index.ts

```diff
@@ -128,7 +128,7 @@ export const defaultAdminProductFields: (keyof Product)[] = [
   "updated_at",
   "deleted_at",
   "metadata",
-]
+] as unknown as (keyof Product)[]
```

Comment:
This might not be necessary. Adding a partial at 105 might make it work

Reply:
yeah it can be removed